### PR TITLE
added a fix to GenericPollingScheduledTask

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-40734

## Description
Added escaping for quotations in arg names and values both in **ScheduleGenericPolling** and **GenericPollingScedulledTask**

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/CommonScripts/Scripts/GenericPollingScheduledTask/GenericPollingScheduledTask.js [hunk 1]
```diff
@@ -103,7 +103,7 @@
     }
     var cmd = '!GenericPollingScheduledTask pollingCommand="' + pollingCommand + '" pollingCommandArgName="' + pollingCommandArgName + '"' + playbookIdStr;
     cmd += ' ids="' + idsStr + '" pendingIds="' + pendingIds.replace(/"/g,'\\"') + '" interval="' + interval + '" timeout="' + (parseInt(timeout) - parseInt(interval)) + '" tag="' + tag + '"';
-    cmd += ' additionalPollingCommandArgNames="' + additionalArgNames + '" additionalPollingCommandArgValues="' + additionalArgValues + '"';
+    cmd += ' additionalPollingCommandArgNames="' + additionalArgNames.replace(/"/g,'\\"') + '" additionalPollingCommandArgValues="' + additionalArgValues.replace(/"/g,'\\"') + '"';
```
[the remark this answers is not in the document]
Comment (ShahafBenYakir) [reply Packs/CommonScripts/Scripts/GenericPollingScheduledTask/GenericPollingScheduledTask.js:106]: No need.

**Comment by ShahafBenYakir** [comment]
@tkatzir I added you so you can check the part related to sanitization that was recently added.

**Review by tkatzir — APPROVED** [review]
Great!
Approved, with minor suggestions re RN.

File: Packs/CommonScripts/ReleaseNotes/1_15_54.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+#### Scripts
+
+##### ScheduleGenericPolling
+
+- Fixed an issue where the script failed when given values containing quotations.
+- Updated the Docker image to: *demisto/python3:3.11.9.107902*.
+##### GenericPollingScheduledTask
+
+- Fixed an issue where the script failed when given values containing quotations.
```
[the remark this answers is not in the document]
Comment (ShirleyDenkberg) [reply Packs/CommonScripts/ReleaseNotes/1_15_54.md:10]: Looks good

File: Packs/CommonScripts/ReleaseNotes/1_15_54.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+#### Scripts
+
+##### ScheduleGenericPolling
+
+- Fixed an issue where the script failed when given values containing quotations.
```
[the remark this answers is not in the document]
Comment (ShirleyDenkberg) [reply Packs/CommonScripts/ReleaseNotes/1_15_54.md:6]: Looks good

**Pull request merged**